src/db/models/governingEntity.ts

```diff
@@ -12,3 +15,25 @@ export type GoverningEntityId = Brand<
 export const GOVERNING_ENTITY_ID = brandedType<number, GoverningEntityId>(
   t.number
 );
+
+export default defineIDModel({
+  tableName: 'governingEntity',
+  fields: {
+    generated: {
+      id: {
+        kind: 'branded-integer',
+        brand: GOVERNING_ENTITY_ID,
+      },
+    },
+    optional: {},
+    required: {
+      planId: { kind: 'branded-integer', brand: PLAN_ID },
```

Comment:
This should be under `accidentallyOptional`, it's nullable (according to the SQL schema).